app/invocation/invocation_suggestion_card.tsx

```diff
@@ -70,6 +71,108 @@ const matchers: SuggestionMatcher[] = [
+      }
+    }
+    if (!flags.length) return null;
+    return {
+      level: SuggestionLevel.INFO,
+      message: (
+        <>
+          This build is using the old BuildBuddy endpoint, <span className="inline-code">cloud.buildbuddy.io</span>.
+          Migrate to <span className="inline-code">remote.buildbuddy.io</span> for improved performance.
+        </>
+      ),
+      reason: (
+        <>
+          Shown because this build has the effective flag{flags.length === 1 ? "" : "s"}{" "}
+          {flags.map((flag, i) => (
+            <>
+              <span className="inline-code">
+                --{flag}={model.optionsMap.get(flag)}
+              </span>
+              {i < flags.length - 1 && " "}
+            </>
+          ))}
+        </>
+      ),
+    };
+  },
+  // Suggest using cache compression
+  ({ model }) => {
+    if (!capabilities.config.expandedSuggestionsEnabled) return null;
+
+    if (model.optionsMap.get("experimental_remote_cache_compression")) return null;
+    if (!model.optionsMap.get("remote_cache") && !model.optionsMap.get("remote_executor")) return null;
+    const version = model.started?.buildToolVersion;
+    if (!version) return null;
+    const segments = version.split(".").map(Number);
+    // Bazel pre-v5 doesn't support compression.
+    if (!segments[0] || segments[0] < 5) return null;
+
+    return {
+      level: SuggestionLevel.INFO,
+      message: (
+        <>
+          Consider adding the Bazel flag <span className="inline-code">--experimental_remote_cache_compression</span> to
+          improve remote cache throughput.
+        </>
+      ),
+      reason: <>Shown because this build is cache-enabled but compression is not enabled.</>,
+    };
+  },
+  // Suggest using --jobs
+  ({ model }) => {
+    if (!capabilities.config.expandedSuggestionsEnabled) return null;
+
+    if (!model.optionsMap.get("remote_executor")) return null;
+    if (model.optionsMap.get("jobs")) return null;
+
+    return {
+      level: SuggestionLevel.INFO,
+      message: (
+        <>
+          Configuring a jobs count will allow more actions to execute in parallel, which can significantly improve
+          remote execution performance. We recommend starting with <span className="inline-code">--jobs=50</span> and
+          working your way up.
+        </>
+      ),
+      reason: <>Shown because this build has remote execution enabled, but a jobs count is not configured.</>,
+    };
+  },
+  // Suggest using remote_download_minimal
+  ({ model }) => {
+    if (!capabilities.config.expandedSuggestionsEnabled) return null;
+
+    if (!model.optionsMap.get("remote_cache") && !model.optionsMap.get("remote_executor")) return null;
+    if (model.optionsMap.get("remote_download_outputs")) return null;
+
+    return {
+      level: SuggestionLevel.INFO,
+      message: (
+        <>
+          Adding the Bazel flag <span className="inline-code">--remote_download_minimal</span> can significantly improve
```

Comment:
you might want to suggest (and look for) either --remote_download_minimal or --remote_download_toplevel  (the latter is needed instead in some scenarios)

Reply:
Done